src/capture/pcap_service.h

```diff
@@ -105,6 +105,15 @@ int dump_file_pcap(struct pcap_context *ctx, char *file_path,
  */
 int inject_pcap(struct pcap_context *ctx, uint8_t *packet, size_t size);
 
+/**
+ * @brief Return pcap capture statistics
+ *
+ * @param ctx The pcap context
+ * @param ps The pcap_stat structure
```

Comment:
Can you add `[in]` and `[out]` to these params? I don't know which is the input and which is the output.

(Adding `const` to the input param if possible will be nice too)

Reply:
Done